Makefile

```diff
@@ -305,9 +317,9 @@ util/build_version.cc: FORCE
 	else mv -f $@-t $@; fi
 endif
 
-LIBOBJECTS = $(LIB_SOURCES:.cc=.o)
-LIBOBJECTS += $(TOOL_LIB_SOURCES:.cc=.o)
-MOCKOBJECTS = $(MOCK_LIB_SOURCES:.cc=.o)
+LIBOBJECTS = $(LIB_SOURCES:.cc=.cc.o) $(LIB_SOURCES_C:.c=.c.o) $(LIB_SOURCES_ASM:.S=.S.o)
```

Comment:
Can you explain the ration behind changing from ".o" to ".cc.o"?
I saw in the build script both were built, so it's likely we have missed something.